frontend/src/types.ts

```diff
@@ -683,13 +683,15 @@ export interface TrendResultWithAggregate extends TrendResult {
 
 export interface FunnelStep {
     action_id: string
-    average_conversion_time: number
+    average_conversion_time: number | null
     count: number
     name: string
     order: number
     people: string[]
     type: EntityType
     labels?: string[]
+    breakdown?: FunnelStep[]
```

Comment:
By making this a recursive type, we can construct an array where the the top-level objects contain count and step metadata for all breakdown values, and the children describe subsets of that step.